microprofile/config-smallrye/src/main/java/org/wildfly/extension/microprofile/config/smallrye/MicroProfileConfigExtension.java

```diff
@@ -55,7 +55,8 @@
     private static final String RESOURCE_NAME = MicroProfileConfigExtension.class.getPackage().getName() + ".LocalDescriptions";
 
     protected static final ModelVersion VERSION_1_0_0 = ModelVersion.create(1, 0, 0);
-    private static final ModelVersion CURRENT_MODEL_VERSION = VERSION_1_0_0;
+    protected static final ModelVersion VERSION_1_1_0 = ModelVersion.create(1, 1, 0);
+    private static final ModelVersion CURRENT_MODEL_VERSION = VERSION_1_1_0;
```

Comment:
Why minor? We have been doing a major bump for major WildFly versions.

Reply:
I thought about this and I'm not sure that change requires a major bump. I'm not modifying the management model in a significant way: there is only 1 flag changed on a single attribute.
I thinks this bug fix fits a minor version bump but if we have required a major bump no matter the changes, I can amend my PR.

Reply:
The basic reasons we do major version bumps is:

1) To ensure we always have space for new API versions downstream, in whatever downstream variants we may care about.
2) KISS principle.

You have a really strong sense of possible downstream needs so I'm happy to defer to your judgement.

Reply:
I'm fine with keeping a minor bump for that single change... If there were any other changes done to this subsystem before WildFLy 21 is released, then I'll make major bump at that time